static/docs/chak.md

```diff
@@ -55,14 +55,14 @@ Other game-end conditions result in a draw:
 ### King (K)
 ![King](https://github.com/gbtami/pychess-variants/blob/master/static/images/ChakGuide/ChakKing.png)
 
-The King (*Ajaw*, which rhymes with Dachau) moves like a king in most classic forms of chess: one square in any direction. However, what is unique about the king in Chak is that it can promote upon reaching the river.
+The King (*Ajaw*, which rhymes with Dachau) moves like a king in most classic forms of chess: one square in any direction. However, the king in Chak is unique in that it can promote upon reaching the river.
 
 ### Divine King (D)
 ![Divine King](https://github.com/gbtami/pychess-variants/blob/master/static/images/ChakGuide/DivineKing.png)
 
-The Divine King moves up to *two* squares in any direction. It can also pass through squares that are threatened by the opponent’s pieces (in other words, it is not like the Western Chess king that cannot castle through check). Remember that as a promoted piece, it cannot go back beyond the river! The divine king has the ability to end the game by reaching the altar without moving into check. Because of its range, the divine king has the ability to deliver checkmate on an unpromoted king.
+The Divine King moves up to *two* squares in any direction. It can also pass through squares that are threatened by the opponent’s pieces (in other words, it is not like the Western Chess king that cannot castle through check). Remember that as a promoted piece, it cannot go back beyond the river! The divine king has the ability to end the game by reaching the altar without moving into check. Because of its range, the divine king has the ability to deliver checkmate to an unpromoted king.
 
-Note that the symbol used only differs slightly from the king’s. They are meant to be the same, but the slight difference is to help players see the promotion and also for the two types of king to be distinct on the board editor.
+Note that the symbol used only differs slightly from the king’s. They are meant to be the same, but the slight difference is to help players see the promotion and also for the two types of kings to be distinct on the board editor.
```

Comment:
This doesn't sound correct to me. I know for sure the king word should be singular. There is only a single king piece on each side, but he can promote to a different type.

Maybe "two **forms** of **the king**"